### PR TITLE
fix: use a svelte-notifications

### LINKED ISSUE #31: Migrate to Svelte Notifications
Now that our package Svelte Notifications is ready, it would be great to use it
over in-house implementation. Lets seek for improvements!

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

File: src/lib/components/Notification/Notification.svelte [hunk 1]
```diff
@@ -1,17 +1,21 @@
 <script lang="ts">
+  import { NotificationKind } from '@whizzes/svelte-notifications';
+  import type { Notification } from '@whizzes/svelte-notifications'
```
Comment (LeoBorai) [inline src/lib/components/Notification/Notification.svelte]: Please rearrange imports

File: src/lib/components/Notification/Notification.svelte [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 
-  import type { Notification } from '$lib/stores/notification';
+  // import notificationStore, {
+  //   NotificationKind
+  // } from '$lib/stores/notification';
 
-  export let notification: Notification;
+  // import type { Notification } from '$lib/stores/notification';
+
+  export let notification:Notification
```
Comment (LeoBorai) [inline src/lib/components/Notification/Notification.svelte]: Remove commented

File: src/lib/components/Notification/Notification.svelte [hunk 3]
```diff
@@ -51,10 +55,10 @@
...
     <p>{notification.message}</p>
   </div>
-  <button
+  <!-- <button
     on:click={() => notificationStore.close(notification.id)}
     class="flex justify-center items-center h-[24px] w-[24px] p-1 rounded-full hover:text-gray-400 cursor-pointer"
   >
     <Close class="h-6 w-6" />
-  </button>
+  </button> -->
```
Comment (LeoBorai) [inline src/lib/components/Notification/Notification.svelte]: Remove commented

File: src/routes/(auth)/login/+page.svelte [hunk 4]
```diff
@@ -6,9 +6,9 @@
   import TextField from '$lib/components/TextField.svelte';
   import Card from '$lib/components/Card.svelte';
   import Button from '$lib/components/Button.svelte';
-  import notification from '$lib/stores/notification';
   import { LoginError, type ErrorMessages } from './shared';
   import { UserErrorCode } from '$lib/graphql/schema';
+  import { notifications } from '@whizzes/svelte-notifications';
```
Comment (LeoBorai) [inline src/routes/(auth)/login/+page.svelte]: Rearrange imports please

**Review by LeoBorai — CHANGES_REQUESTED** [review]

File: src/lib/components/Notification/Notification.svelte [hunk 5]
```diff
@@ -1,17 +1,15 @@
 <script lang="ts">
+  import { NotificationKind } from '@whizzes/svelte-notifications';
   import classNames from 'classnames';
-
+
```
Comment (LeoBorai) [inline src/lib/components/Notification/Notification.svelte]:
```suggestion

```

File: src/routes/(auth)/signup/+page.svelte [hunk 6]
```diff
@@ -5,7 +5,7 @@
   import TextField from '$lib/components/TextField.svelte';
   import Card from '$lib/components/Card.svelte';
   import Button from '$lib/components/Button.svelte';
-  import notification from '$lib/stores/notification';
+  import { notifications } from '@whizzes/svelte-notifications';
```
Comment (LeoBorai) [inline src/routes/(auth)/signup/+page.svelte]: Rearrange imports

File: src/routes/+layout.svelte [hunk 7]
```diff
@@ -4,9 +4,10 @@
   import { createClient, setContextClient } from '@urql/svelte';
   
   import { page } from '$app/stores';
-  import NotificationsList from '$lib/components/Notification/NotificationsList.svelte';
+  import {NotificationList, Position} from '@whizzes/svelte-notifications'
```
Comment (LeoBorai) [inline src/routes/+layout.svelte]: Rearrange imports

File: src/routes/+layout.svelte [hunk 8]
```diff
@@ -4,9 +4,10 @@
...
   
   import '../app.css';
+  import Notification from '$lib/components/Notification/Notification.svelte';
```
Comment (LeoBorai) [inline src/routes/+layout.svelte]: Rearrange imports

File: src/routes/+layout.svelte [hunk 9]
```diff
@@ -53,4 +54,6 @@
 </script>
 
 <slot />
-<NotificationsList />
+<NotificationList class="fixed top-0 right-0 mt-[54px] p-6 space-y-4 overflow-y-auto max-h-[calc(100vh-54px)]" let:notification position={Position.TopRight}>
```
Comment (LeoBorai) [inline src/routes/+layout.svelte]: Why use both?

- `position={Position.TopRight}`
- `fixed top-0 right-0`

**Review by LeoBorai — CHANGES_REQUESTED** [review]

File: src/routes/+layout.svelte (hunk 9 above)
Comment (Michael-Liendo) [reply src/routes/+layout.svelte]: Using `Position={Position.TopRight}` does not work for some reason. Although you could remove `Position` and leave the custom classes

**Review by LeoBorai — APPROVED** [review]
LGTM!

**Pull request merged**